build.sbt

```diff
@@ -568,9 +568,9 @@ lazy val protobufSettings = Def.settings(
   libraryDependencies ++= {
     val scalapbVersion =
       if (isScala211.value) "0.9.8"
-      // freeze versions so that we don't depend on newer version, scalapb 0.11.13 depends on Scala 2.13.10 etc.
-      else if (scalaVersion.value == "2.13.11") "0.11.13"
-      else if (scalaVersion.value == "2.13.12" || scalaVersion.value == "2.13.13") "0.11.15"
+      // for SIP-51, freeze version to the latest ScalaPB built against the earliest Scala 2.13.x version we support
+      else if (EarliestScala213 == "2.13.11") "0.11.13"
+      else if (EarliestScala213 == "2.13.12" || scalaVersion.value == "2.13.13") "0.11.15"
```

Comment:
this line is dead code at the moment, but I kept it since it will be useful as EarliestScala213 moves forward